components/connection-token-handler/pkg/controller/tokenrequest/tokenrequest_controller.go

```diff
@@ -5,7 +5,7 @@ import (
 	"log"
 	"time"
 
-	connectorservicev1alpha1 "github.com/kyma-project/kyma/components/connection-token-handler/pkg/apis/connectorservice/v1alpha1"
+	connectorservicev1alpha1 "github.com/kyma-project/kyma/components/connection-token-handler/pkg/apis/applicationconnector/v1alpha1"
```

Comment:
Please change the dependency name `connectorservicev1alpha1`

Reply:
Done!